stable/sealed-secrets/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "sealed-secrets.fullname" . }}
+  labels:
+    app: {{ template "sealed-secrets.fullname" . }}
+    chart: {{ template "sealed-secrets.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "sealed-secrets.fullname" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "sealed-secrets.fullname" . }}
+        release: {{ .Release.Name }}
+    spec:
+      {{ if .Values.rbac.create }}
+      serviceAccountName: {{ template "sealed-secrets.serviceAccountName" . }}
```

Comment:
@unguiculus Is this okay to leave here? I didn't want to enforce the use of `serviceAccountName` in the case you aren't using RBAC resources because it is unlikely you will need the service account at all. Is this an acceptable practice?

Reply:
Remove the `if` statement. If you consider my other comment, the default service account is used if nothing is configured.